### PR TITLE
IWF-129: Add executingStateIdMode

### PR DESCRIPTION
**Pull request by lwolczynski**

### CODE REVIEW

File: iwf-sdk.yaml [hunk 1]
```diff
@@ -1329,8 +1329,14 @@ components:
       properties:
         disableSystemSearchAttribute: # deprecated in favor of enabledExecutingStateExecutionIds
           type: boolean
-        enabledExecutingStateExecutionIds:
+        enabledExecutingStateExecutionIds: # deprecated in favor of executingStateIdMode
```
Comment (longquanzheng) [inline iwf-sdk.yaml]: It’s never being used. We can safely remove it

Comment (lwolczynski) [reply iwf-sdk.yaml]: Done now

Comment (lwolczynski) [reply iwf-sdk.yaml]: Should I keep it in `iwf.yaml`?

Comment (lwolczynski) [reply iwf-sdk.yaml]: Doesn't seem to be used either. Removed

**Review by longquanzheng — APPROVED** [review]

**Pull request merged**